src/plib/resources/locales/en-US.php

```diff
@@ -45,28 +45,28 @@
-    'controllers.system.panelSecured' => 'Plesk is secured with a valid SSL certificate',
-    'controllers.system.panelNotSecured' => 'Plesk is not secured with a valid SSL certificate',
+    'controllers.system.panelSecured' => 'Plesk is secured with a valid SSL/TLS certificate',
+    'controllers.system.panelNotSecured' => 'Plesk is not secured with a valid SSL/TLS certificate',
     'controllers.system.secureDesc' => 'Secure Plesk with Let\'s Encrypt',
-    'controllers.system.letsencryptDesc' => 'Let\'s Encrypt is a certificate authority (CA) that allows you to create free SSL certificates for your domains.',
+    'controllers.system.letsencryptDesc' => 'Let\'s Encrypt is a certificate authority (CA) that allows you to create free SSL/TLS certificates for your domains.',
     'controllers.system.letsencryptInstall' => 'Install Let\'s Encrypt and secure Panel with free SSL certificate',
     'controllers.system.http2Desc' => 'HTTP/2 improves performance; specifically, end-user perceived latency, network and server resource usage.',
     'controllers.system.http2Enabled' => 'HTTP/2 is enabled',
     'controllers.system.http2Button' => 'Enable HTTP/2',
     'controllers.system.nginxDesc' => 'Enable nginx so that HTTP/2 can be turned on. HTTP/2 improves performance; specifically, end-user perceived latency, network and server resource usage.',
     'controllers.system.nginxButton' => 'Enable nginx reverse proxy and HTTP/2',
-    'controllers.system.aiDesc' => 'Install nginx to make HTTP/2 available. To install nginx, select nginx web server in Web hosting group in Plesk Installer. HTTP/2 improves performance; specifically, end-user perceived latency, network and server resource usage.',
+    'controllers.system.aiDesc' => 'Install nginx to make HTTP/2 available. To install nginx, select nginx web server in the Web hosting group in Plesk Installer. HTTP/2 improves performance; specifically, end-user perceived latency, network and server resource usage.',
     'controllers.system.aiButton' => 'Install nginx web server and enable HTTP/2',
     'controllers.system.datagridDesc' => 'The Datagrid scanner analyzes your server configuration and compares it to real world results from servers around the world to report reliability and security vulnerabilities.  On top of that, it\'s free.',
     'controllers.system.datagrid' => 'Datagrid reliability and vulnerability scanner',
     'controllers.system.datagridActivate' => 'Activate the Datagrid reliability and vulnerability scanner',
     'controllers.system.datagridInstall' => 'Install the Datagrid reliability and vulnerability scanner',
-    'controllers.system.patchmanDesc' => 'Patchman automatically and safely patches vulnerabilities in CMSs like WordPress, Joomla and Drupal. On top of that, it cleans up malware.',
+    'controllers.system.patchmanDesc' => 'Patchman automatically and safely patches vulnerabilities in CMSs like WordPress, Joomla, and Drupal. On top of that, it cleans up malware.',
```

Comment:
This is a rule of "serial comma" in English punctuation: https://en.wikipedia.org/wiki/Serial_comma